creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -456,59 +455,36 @@ class SnapbackSM {
   }
 
   /**
-   * Depending on the size of `unhealthyReplicas`:
-   * 1. Determine a new replica set
-   * 2. Write new replica set to URSM
-   * 3. Sync data to new replica set
+   * 1. Write new replica set to URSM
+   * 2. Sync data to new replica set
    *
    * @param {number} userId user id to issue a reconfiguration for
    * @param {string} wallet wallet address of user id
    * @param {string} primary endpoint of the current primary node on replica set
    * @param {string} secondary1 endpoint of the current first secondary node on replica set
    * @param {string} secondary2 endpoint of the current second secondary node on replica set
-   * @param {string[]} unhealthyReplicas array of endpoints of current replica set nodes that are unhealthy
-   * @param {string[]} healthyNodes array of healthy Content Node endpoints used for selecting new replica set
-   * @param {Object} replicaSetNodesToUserClockStatusesMap map of secondary endpoint strings to (map of user wallet strings to clock value of secondary for user)
+   * @param {Object} newReplicaSet {newPrimary, newSecondary1, newSecondary2, issueReconfig, reconfigType}
    */
   async issueUpdateReplicaSetOp(
     userId,
     wallet,
     primary,
     secondary1,
     secondary2,
-    unhealthyReplicas,
-    healthyNodes,
-    replicaSetNodesToUserClockStatusesMap
+    newReplicaSet
   ) {
-    this.log(
```

Comment:
no concerns, just curious why you removed this log

Reply:
must've been an accident when I was changing the phase stuff. I'll add that back. it's been helpful for me to debug how long this function takes

Reply:
oh actually I removed it because the function doesn't need healthyReplicas unhealthyReplicas anymore so there wasn't anything to log. I'll just add it back to log the user and wallet

Reply:
sg